converter/config/config.go

```diff
@@ -77,6 +77,13 @@ type RITA interface {
 	GetConnectionConfig() MongoDBConnection
 	GetDBRoot() string
 	GetMetaDB() string
+	GetStrobe() Strobe
+}
+
+//Strobe contains configuration for populating the
+//freqConn collection / Strobes analysis in RITA
+type Strobe interface {
+	GetConnectionLimit() int
 }
```

Comment:
Generally had to touch a few files to implement the ConnectionLimit config option for strobes as in RITA. 

Changed: 
converter/config/config.go
converter/config/yaml/output.go
converter/integrationtest/config.go
converter/etc/config.yaml
runtime/etc/converter/converter.yaml